lib/mods/replace-constructors.js

```diff
@@ -3,6 +3,30 @@
 const nodeTypes = require('ts-morph').TypeGuards;
 const globalsSet = require('../globals-set.js');
 
+/**
+ * Adds an import for the module to the list
+ *
+ * @param {string} mod - the name of the module to import from
+ * @param {string} original - the name of the original to get
+ * @param {Object} imports - the current list of imports
+ */
+function updateImports(mod, original, imports) {
+  if (mod in imports) {
+    if (original in imports[mod]) {
+      // if an import of original from module exists, noop
+      return;
+    }
+    // if an import from module exists without importing original,
+    // add original to the import
+    imports[mod].append(original);
```

Comment:
There is no append() method on arrays. The fact that this slipped through means the tests are not covering this.

If this code path isn't testable currently, then, we should remove it until we add some tests that exercise it.

Reply:
Yup, `.append()` is python, good catch. I'll write up a test for these cases

Reply:
Hmm, currently the code only gets the originals for constructors, meaning this case won't come up until a mod for method calls or statics is written.

Reply:
Best to delete any non-used code for now, then. I suspect it will change in the future.